core/src/test/scala/org/apache/spark/deploy/master/MasterSuite.scala

```diff
@@ -264,16 +265,17 @@ class MasterSuite extends SparkFunSuite
     val conf = new SparkConf()
     val localCluster = new LocalSparkCluster(2, 2, 512, conf)
     localCluster.start()
+    val masterUrl = s"http://localhost:${localCluster.masterWebUIPort}"
```

Comment:
Similar to https://github.com/apache/spark/pull/31344, Not easy to maintain the same format as the master, so added masterUrl(line 268)